### PR TITLE
Fix config_management/README.md

### PR DESCRIPTION
**Pull request by brb**
* Some cleanup.
* Fix task file names.
* Use `ansible_inventory`.

cc @marccarre

### CODE REVIEW

File: config_management/README.md [hunk 1]
```diff
@@ -70,38 +70,43 @@ and finally run:
...
 ```
 ansible-playbook --private-key=$vagrant_ssh_id_file -u $vagrant_ssh_user \
 --ssh-extra-args="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null" \
--i "$vagrant_ssh_host:$vagrant_ssh_port," setup_docker_k8s_weave-kube.yml \
+-i "$vagrant_ssh_host:$vagrant_ssh_port," setup_weave-kube.yml \
 --extra-vars "docker_version=1.12.3 kubernetes_version=1.4.4"
 ```
 
+NOTE: Kubernetes APT repo includes only the latest version, so currently
+retrieving an older version will fail.
```
Comment (marccarre) [inline config_management/README.md]: Very good point, given you and Bryan have already had issues with this.

File: config_management/README.md [hunk 2]
```diff
@@ -4,8 +4,8 @@
 
 This project allows you to configure a machine with:
 
-* Docker and Weave Net: `setup_docker_weave-net.yml`
-* Docker, Kubernetes and Weave Kube (CNI plugin): `setup_docker_k8s_weave-kube.yml`
+* Docker and Weave Net: `setup_weave-net.yml`
```
Comment (marccarre) [inline config_management/README.md]: This should really be:

```
* Docker and Weave Net for development: `setup_weave-net_dev.yml`
* Docker and Weave Net for testing: `setup_weave-net_test.yml`
```
and then line 8 with `setup_weave-kub.yml`.

**Comment by brb** [comment]
PTAL.

**Comment by brb** [comment]
PTAL @marccarre

**Comment by marccarre** [comment]
LGTM, thank you very much!

**Pull request merged**